pkg/kipcompute/compute.go

```diff
@@ -123,13 +123,15 @@ func replaceIP(projectID string, zone string, instance string, pool string, conf
 		logrus.WithFields(logrus.Fields{"pkg": "kubeip", "function": "replaceIP"}).Errorf("Instance not found %s zone %s: %q", instance, zone, err)
 		return err
 	}
-	accessConfigName := inst.NetworkInterfaces[0].AccessConfigs[0].Name
-	op, err := computeService.Instances.DeleteAccessConfig(projectID, zone, instance, accessConfigName, "nic0").Do()
-	if err != nil {
-		logrus.Errorf("DeleteAccessConfig %q", err)
-		return err
+	if len(inst.NetworkInterfaces) > 0 && len(inst.NetworkInterfaces[0].AccessConfigs) > 0 {
```

Comment:
A VM always have a network interface so the first check is redundant. 

Regardin accssConfig - from the docs:
`if there are no accessConfigs specified, then this instance will have no external internet access.`

We can leave this check however a VM without extenal internet access is not the use case for kubeip.